index.html

```diff
@@ -2458,14 +2458,24 @@ <h3>Data Schemas</h3>
 
         <dl>
           <dt><var>credentialSchema</var></dt>
-          <dd>
-The value of the <code>credentialSchema</code> <a>property</a> MUST be one or
-more data schemas that provide <a>verifiers</a> with enough information to
-determine if the provided data conforms to the provided schema. Each
-<code>credentialSchema</code> MUST specify its <code>type</code> (for example,
-<code>JsonSchemaValidator2018</code>), and an <code>id</code> <a>property</a>
-that MUST be a <a>URL</a> identifying the schema file. The precise contents of
-each data schema is determined by the specific type definition.
+          <dd><p>
+The value of the <code>credentialSchema</code> <a>property</a> MUST be a single
+data schema object which provides <a>verifiers</a> with enough information 
+to determine if the provided data conforms to the provided schema object. Each
```

Comment:
"Each" suggests multiple schemas?

Reply:
Yes, confused by this, and your other comments... I think we should start with 1, since thats closest to what we had in 1.1

Reply:
Sorry, had a hard time with this language. My intention is to say the "credentialSchema" property can only list one, though the value it resolves to can list 1 or many.

Reply:
I'm still having troubles parsing what you are saying.

are you saying this should be valid

```
"credentialSchema": [{"type": "CredentialSchema2022", ...}, {"type": "JsonSchemaValidator2018", ...}]
```

or this

```
"credentialSchema": {"type": ["CredentialSchema2022", "JsonSchemaValidator2018"], ...}
```

or something else entirely?

Reply:
neither. I'm saying

```
"credentialSchema": {"type": ["CredentialSchema2022"]}"
```

and the value of the `id` for `CredentialSchema2022` can contain multiple schemas

Reply:
how can an `id` contain multiple things? The document that resolves from that `id` can have multiple schemas?

Reply:
yes, the document that resolves from the `id` can have multiple schemas as defined by the type

Reply:
```suggestion
to determine whether the provided data conforms to the provided schema object. Each
```